src/main/scala/encry/local/miner/Miner.scala

```diff
@@ -47,12 +49,7 @@ class Miner extends Actor with Logging {
   def needNewCandidate(b: EncryBlock): Boolean =
     !candidateOpt.flatMap(_.parentOpt).map(_.id).exists(_.sameElements(b.header.id))
 
-  def shouldStartMine(b: EncryBlock): Boolean =
-    settings.node.mining && candidateOpt.forall(_.timestamp <= b.header.timestamp)
-
-  def unknownMessage: Receive = {
-    case m => logWarn(s"Unexpected message $m")
-  }
+  override def receive: Receive = if (settings.node.mining) miningEnabled else miningDisabled
```

Comment:
Maybe we should change condition to `settings.node.mining && syncingDone`, so we don't start mining until we are synced?